extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -78,7 +78,7 @@ export class AmpAd extends AMP.BaseElement {
       if (!a4aRegistry[type] ||
           (!(adConfig[type] || {}).remoteHTMLDisabled &&
           this.win.document.querySelector('meta[name=amp-3p-iframe-src]') &&
-          !this.win.document.getElementById('amp-rtc')) ||
+          !this.win.document.getElementById('rtc-config')) ||
```

Comment:
I worry that these checks are getting longer than we intended... we should consider moving into the predicates at some point.  Let's add a TODO.

Reply:
done